src/components/PostItem.tsx

```diff
@@ -154,6 +161,19 @@ export default function PostItem({
           ))}
         </div>
       )}
+      {
+        post.reactions && <div className="action-bar" style={{flexWrap: 'wrap'}}>
+          {
+            post.reactions?.map((val, index) => <div key={index} className="emoji-reaction" style={{
+              backgroundColor: '#2a2c38', margin: '0.5rem', padding: '0.3rem', display: 'flex', alignItems: 'center', borderRadius: '5px'
+            }}>
```

Comment:
I would appreciate if these were moved to the theme files to keep the application easy to customize.